openpype/modules/deadline/plugins/publish/submit_publish_job.py

```diff
@@ -270,6 +270,7 @@ def _submit_deadline_post_job(self, instance, job, instances):
                 "Version": self.plugin_pype_version,
                 "Arguments": " ".join(args),
                 "SingleFrameOnly": "True",
+                "RenderSetupIncludeLights": None  # noqa
```

Comment:
I still think this line shouldn't be here at all. `RenderSetupIncludeLights` isn't a Plugin setting for the Deadline OpenPype Plugin.
```suggestion
```

Reply:
Fixed!